sound/soc/sof/intel/hda-dai.c

```diff
@@ -718,6 +796,18 @@ void hda_set_dai_drv_ops(struct snd_sof_dev *sdev, struct snd_sof_dsp_ops *ops)
 {
 	int i;
 
+	if (sdev->dspless_mode_selected) {
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
```

Comment:
didn't I change this in my HDaudio cleanups? I don't have the code handy but I thought I changed that to CONFIG_SND_SOC_SOF_HDAUDIO_CODEC

Reply:
oh, yes you did, but it still works :o